lib/ActionLibrary/source/Library/Collision/Collider.cpp

```diff
@@ -0,0 +1,39 @@
+#include "Library/Collision/Collider.hpp"
+#include "Library/Collision/HitInfo.hpp"
+
+namespace al {
+    Collider::Collider(CollisionDirector *pCollisionDirector, const sead::Matrix34f *pBaseMtx, const sead::Vector3f *pTrans, const sead::Vector3f *pGravity, f32 radius, f32 offsetY, u32 numHitInfos) :
+            mCollisionDirector(pCollisionDirector),
+            mBaseMtx(pBaseMtx),
+            mTrans(pTrans),
+            mGravity(pGravity),
+            mRadius(radius),
+            mOffsetY(offsetY),
+            _48(numHitInfos),
+            _58(sead::Vector3f(0.0f, 0.0f, 0.0f)),
```

Comment:
These are most likely implicitly constructed to 0. Can we see if removing these still matches?

Reply:
assuming you're referring to _58 and _64: nope, they do get stored to twice. removing them causes it to not match

Reply:
Sounds good then.